internal/cmd/run/cypress.go

```diff
@@ -98,6 +99,10 @@ func runCypress(cmd *cobra.Command, tc testcomposer.Client, rs resto.Client, as
 	}
 	cypress.SetDefaults(&p)
 
+	if p.Sauce.Tunnel.ID != "" {
```

Comment:
🍝 

Why here? Is it really the responsibility of `run/cypress.go` to handle the deprecation of **config** fields? `cypress.FromFile()` reads the config file, `cypress.SetDefaults()` set the defaults. Aren't those two more appropriate?

Reply:
I'll move it to `FromFile`

Reply:
My original thoughts was `FromFile` is only reading cfg files and `SetDefaults` is setting default values for the config. Seems neither of them is related to update the value for a flag 🤔